app/assets/stylesheets/_category.scss

```diff
@@ -0,0 +1,13 @@
+.category {
+  padding-left: 10px;
+  padding-right: 10px;
+  list-style-type: none;
+  margin-left: 15px;
+  margin-right: 15px;
+
+}
+
+.category :hover {
+  color: #bfa24b;
```

Comment:
Color literals like `#bfa24b` should only be used in variable declarations; they should be referred to via variable everywhere else.